features/test/kata.feature

```diff
@@ -1,5 +1,16 @@
 Feature: example kata container scenarios
   @admin
   @destructive
+  @upgrade-prepare
   Scenario: kata container operator installation
+    Given the master version >= "4.6"
     Given kata container has been installed successfully in the "kata-operator" project
+    And I verify kata container runtime is installed into the a worker node
+
+  @admin
+  @destructive
+  @upgrade-prepare
+  Scenario: test delete kata installation
+    Given the master version >= "4.6"
+    Given I switch to cluster admin pseudo user
+    And I remove kata operator from "kata-operator" namespace
```

Comment:
Not sure if I read this correctly. But a successful deletion of the kataconfig should go like this: 1. delete kataconfig example-kataconfig CR (oc delete kataconfig example-kataconfig), this should block until the example-kataconfig is removed and it should also block until the worker machine-config-pool is updated  2. delete the kata-operator namespace (oc delete ns kata-operator).

Reply:
@jensfr that's what I'm doing within the step.

Reply:
@jensfr https://github.com/openshift/verification-tests/pull/1653/files/c0735ba128eb7726582a3e66227c0f1d8538095a#diff-119f8ac5278d90b899ce7a06bcdb073301fab26275a2d44b24964ade734a8eb1R51-R58